### DISCUSSION TITLE
上传文件报错

### DISCUSSION THREAD

**Discussion by SYZhang0805** [houtianze/bypy#664] [Q&A]
<E> [16:31:08] Error accessing 'https://pcs.baidu.com/rest/2.0/pcs/file'
<E> [16:31:08] Function: __rapidupload_file_act
<E> [16:31:08] Website parameters: {'method': 'rapidupload', 'path': '/apps/bypy/avhqv2-64x64.zip', 'content-length': 198612238, 'content-md5': 'e9847206b897882c6d03089d9e590997', 'slice-md5': '27d364ba6a6f88103e73b87976634a08', 'content-crc32': '0xa718ba9', 'ondup': 'overwrite'}
<E> [16:31:08] Full URL: https://pcs.baidu.com/rest/2.0/pcs/file?method=rapidupload&path=%2Fapps%2Fbypy%2Favhqv2-64x64.zip&content-length=198612238&content-md5=e9847206b897882c6d03089d9e590997&slice-md5=27d364ba6a6f88103e73b87976634a08&content-crc32=0xa718ba9&ondup=overwrite&access_token=121.6837e8b9059bf68ade2270f1c5126d3c.Y3hXK5dzjd04vHg6eLxDOlPJuXj_dbrUCOFSTF-.oPaPag
<E> [16:31:08] HTTP Response Status Code: 400
<E> [16:31:08] Error JSON returned:{'error_code': 31023, 'error_msg': 'param error', 'request_id': 7994294746206158305}
Error code: 31023
Error Description: param error
<E> [16:31:08] Website returned: {"error_code":31023,"error_msg":"param error","request_id":7994294746206158305}
<E> [16:31:08] Waiting 10 seconds before retrying...

在上传文件的时候以上错误，请问是怎么回事

**Comment by houtianze** [comment]
不是很确定原因，秒传失败，重试依旧？

**Comment by houtianze** [comment]
谢谢反馈。
这个`'error_code': 31023, 'error_msg': 'param error`貌似是百度那边新的错误，可能是停了或者改了秒传的接口。最新版`1.8.3`解决了这个问题，但是秒传现在已经不可用了。
